src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,160 @@
+            src={omaoedgLookoutViewTelemType}
+            alt="Lookout view usage metric"
+            title="Lookout view usage metric"
+        />
+
+You can change the facet field to anything you want to get more, or less, granularity on your ingest data. For example, changing `usageMetric` to `consumingAccountName` would give you even more detailed information by reporting on ingest by account name, as shown below:
+
+        <img
+            src={omaoedgLookoutViewConsumingAccount}
+            alt="Lookout view consuming account"
+            title="Lookout view consuming account"
+        />
+
+### Install the entity breakdown dashboard (optional) [#install-entity-breakdown-dashboard]
+
+Previously in this tutorial, you installed the ingest baseline dashboard that uses `NrConsumption` as its primary source. But you may find that you also want information on estimating ingest traffic for other events or metrics. If you do, you can install the entity breakdown dashboard, which helps you create other visualizations that use `bytescountestimate()` to help estimate your ingest data. 
+
+To install the entity breakdown dashboard: 
+
+1. Go to the [same quickstart](https://onenr.io/0PoR8zpDYQG) you used for the baseline dashboard.
+2. Select **Install this quickstart**.
+3. Install the dashboard into any account that contains APM, browser monitoring, mobile monitoring, or Kubernetes clusters using the [import dashboard function](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import). You can install this dashboard into multiple accounts, but avoid installing this dashboard into a partnership owner account, or POA.
+
+<Callout variant="tip">
+If you have a parent/child account structure, you can install the dashboard into a parent account and modify the dashboard so you have account-specific charts all in one dashboard.
+</Callout>
+
+4. Select **Done**.
+5. When the quickstart finishes installing, open the **Data governance entity breakdowns** dashboard to access your data.
+
+You can refer back to [this section](/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-baselining#understand-nr-consumption-metrics) to see exactly which event types are used in these breakdowns. Note that these queries consume more resources because they don't work from a pre-aggregated data source like `NrConsumption`, so you may need to adjust the time frames by using additional `WHERE` and `LIMIT` clauses to make them work better in some of your environments.
+
+### Install the cloud integration dashboards (optional) [#install-cloud-integration-dashboards]
+
+New Relic's cloud integrations often give you a significant source of data ingest growth. But without good visualizations, it's often difficult to find where the ingest growth comes from. It's usually because these integrations aren't part of an organization's normal CI/CD pipeline or they aren't part of normal configuration management system.
```

Comment:
Since it's optional, maybe this can go into a collapser. I'm thinking about preventing the illusion of 'more homework'... Could be simple as titling it h3. Optional: Cloud yadda yadda

Reply:
It's possible, but I also wanted to avoid collapser fatigue considering what comes in the following docs. Let's put a pin in this one to revisit next week as well.